### PR TITLE
Added Haunt

### PR DESCRIPTION
**Pull request by jackhill**

### CODE REVIEW

File: list.json [hunk 1]
```diff
@@ -784,6 +784,14 @@
...
     "github": "mkaz/hastie"
   },
+  {
+    "name": "Haunt",
+    "website": "https://dthompson.us/projects/haunt.html",
+    "language": "Scheme",
+    "description": "A simple, functional, hackable static site generator that gives authors the ability to treat websites as Scheme programs.",
+    "license": "GPL-3.0-or-later",
+    "is": "static site generator",
+    "extensible": true
```
Comment (dertuxmalwieder) [inline list.json]: Missing a }, here?

Comment (jackhill) [reply list.json]: Indeed. Thanks, I'll fix that…

Comment (jackhill) [reply list.json]: fixed in 564a57e6d23c63e268e4b91ffe2f0e48dccbf01d

**Comment by dertuxmalwieder** [comment]
Better now. ;-)

**Comment by jackhill** [comment]
Thanks for the quick review and catching my mistake!

**Pull request merged**